### ISSUE TITLE
Fix warning when accessing configurations

### ISSUE THREAD

**Issue by HektorW** [HektorW/vs-code-create-react-folder-component#8]
In `getSubFolders.ts`.

```typescript
const excludedFilesSettings = workspace
    .getConfiguration('files', workspaceFolder)
    .get<ExcludeFilesSetting>('exclude')
```

**Comment by HektorW** [comment]
The warning is:

```
[ms-vscode.csharp] Accessing a resource scoped configuration without providing a resource is not expected. To get the effective value for 'files.exclude', provide the URI of a resource or 'null' for any resource.
```

**Comment by HektorW** [comment]
Should be fixed in v2.0.2
